packages/server/lib/controllers/oauth.controller.ts

```diff
@@ -302,6 +317,10 @@ class OAuthController {
                     additionalAuthParams['code_challenge_method'] = 'S256';
                 }
 
+                if (providerConfig.provider === 'slack' && userScope) {
+                    additionalAuthParams['user_scope'] = userScope;
```

Comment:
Oh interesting that Slack keeps it singular. It's counter intuitive, right?

Reply:
Right, yeah, i kept it singular because Slack kept it singular to match them. Agreed it should be plural but for consistency I went with user_scope